bindings/nodejs/index.d.ts

```diff
@@ -31,7 +31,7 @@ export interface PresignedRequest {
   headers: Record<string, string>
 }
 export class Operator {
-  constructor(scheme: string, options?: Record<string, string> | undefined | null)
+  constructor(scheme: 'azblob' | 'azdfs' | 'cos' | 'dashmap' | 'fs' | 'ftp' | 'ftps' | 'gcs' | 'ghac' | 'hdfs' | 'http' | 'https' | 'ipfs' | 'ipns' | 'ipmfs' | 'memcached' | 'memory' | 'moka' | 'obs' | 'onedrive' | 'gdrive' | 'oss' | 'redis' | 'rocksdb' | 's3' | 'sled' | 'supabase' | 'vercel_artifacts' | 'wasabi' | 'webdav' | 'webhdfs', options: Record<string, string> | undefined | null)
```

Comment:
Thanks for the PR! But I don't like the idea to list all schemes here. How about add a docs link to https://opendal.apache.org/docs/category/services instead?

Reply:
Sorry, I forgot to state the initial purpose of this PR, which is to make the editor provide more precise suggestions and thus reduce the possibility of errors; I agree with you that the current idea is indeed not very good, what about string enums? It gives precise suggestions while providing a clean display

Reply:
I don't want to sync service scheme between different bindings. The benefit seems not worth.

Reply:
And please note that not all services are enabled by default. The service list could wrong or change based on different features.